server/routes/notifications.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createNotification,
+  readNotification,
+  getAllNotifications,
+  getUnreadNotifications,
+} = require("../controllers/notification");
+
+router.route("/create").post(createNotification);
+
+router.route("/read/:id").patch(readNotification);
+
+router.route("/all/:recipient").get(getAllNotifications);
```

Comment:
```suggestion
router.route("/all").get(getAllNotifications);
```
We don't need path parameters for this route and the one below it :)